doc/source/quick_start.rst

```diff
@@ -222,39 +221,22 @@ List files
 
     >>> bucket_name = 'example-mybucket-b2'
     >>> bucket = b2_api.get_bucket_by_name(bucket_name)
-    >>> max_to_show = 1  # max files to show, default=100, optional parameter
-    >>> start_file_name = 'som'  # default is '', optional parameter
-    >>> bucket.list_file_names(start_file_name, max_to_show)
-    {'files': [{'accountId': '451862be08d0',
-       'action': 'upload',
-       'bucketId': '5485a1682662eb3e60980d10',
-       'contentLength': 1870579,
-       'contentSha1': 'd821849a70922e87c2b0786c0be7266b89d87df0',
-       'contentType': 'application/pdf',
-       'fileId': '4_z5485a1682662eb3e60980d10_f1195145f42952533_d20190403_m130258_c002_v0001111_t0002',
-       'fileInfo': {'src_last_modified_millis': '1550988084299'},
-       'fileName': 'som2.pdf',
-       'uploadTimestamp': 1554296578000}],
-     'nextFileName': 'som2.pdf '}
+    >>> for file_info, folder_name in bucket.ls(folder_to_list='', show_versions=False):
+    >>>     print(file_info.file_name, file_info.upload_timestamp, folder_name)
+    f2.txt 1560927489000 None
+    som2.pdf 1554296578000 None
+    some.pdf 1554296579000 None
```

Comment:
it would be nice to see an example where some of the folder names are not `None`